pkg/nats/proxy/constants.go

```diff
@@ -0,0 +1,35 @@
+package proxy
+
+import "fmt"
+
+const (
+	ComputeEndpointSubjectPrefix = "node.compute"
+	CallbackSubjectPrefix        = "node.orchestrator"
+
+	AskForBid       = "AskForBid/1"
```

Comment:
Is the /1 the function arity, or the version or something else?

Reply:
it is the version. I changed the subject names to `AskForBid/v1` for better clarity in https://github.com/bacalhau-project/bacalhau/pull/3188/commits/52bbc1e8da4b7c7827436969d814ff15ec854ba0